vms/avm/service_test.go

```diff
@@ -654,7 +677,20 @@ func TestServiceGetTxJSON_ExportTx(t *testing.T) {
 	"unsignedTx": {
 		"networkID": 10,
 		"blockchainID": "PLACEHOLDER_BLOCKCHAIN_ID",
-		"outputs": null,
+		"outputs": [
+			{
+				"assetID": "2XGxUr7VF7j1iwUp2aiGe4b6Ue2yyNghNS1SuNTNmZ77dPpXFZ",
```

Comment:
I am changing the way the input txs are built. We used to lay them out here and instead I need to build them via `env.txBuilder`.  This will simplify maintenance when dynamic fees will activate and it's the main point of this PR.
All of the other changes in this file around json asserting just follows from the env.txBuilder use